tldraw/apps/tldraw-logseq/src/components/ContextBar/contextBarActionFactory.tsx

```diff
@@ -356,6 +356,7 @@ const SwatchAction = observer(() => {
   return (
     <ColorInput
       title="Color Picker"
+      popoverSide="top"
       color={color}
```

Comment:
`color` could be undefined. Can you fix the types in `ColorInput`?

Reply:
Handled.